packages/mds-attachment-service/service/helpers/index.ts

```diff
@@ -38,7 +38,7 @@ if (env.ATTACHMENTS_BUCKET) {
   /* eslint-disable-next-line */
   aws.config.getCredentials(async err => {
     if (err) {
-      logger.error('Error getting AWS credentials', err.stack || err)
+      logger.error('Error getting AWS credentials', err)
```

Comment:
oh, you really don't want the error.stack in the logs anymore!

Reply:
Heh yeah, the stacktrace is honestly not really very useful outside of debugging in most cases IMO; I've found webpack really munges the symbols, and the JS stack trace where it just says everything is on line 1 is a little misleading 😅